docs/examples/05-job_schedule.jl

```diff
@@ -3,8 +3,8 @@
 # assigns one element to each worker to compute the operation.
 # When the worker is finished, the root sends another element
 # until each element is added 100
-# Inspired on
-# https://www.hpc.ntnu.no/ntnu-hpc-group/vilje/user-guide/software/mpi-and-mpi-io-training-tutorial/basic-mpi/job-queue
+# Inspired on https://www.hpc.ntnu.no/vilje/software/mpi-and-mpi-io-training-tutorial/
+# wget https://www.hpc.ntnu.no/wp-content/uploads/2019/09/mpiexamples.tar.gz
```

Comment:
What is this for?

Reply:
> What is this for?

see text of the closed pull for my initial reasoning and the `tree` command output

the pertinent reference files are not factually viewable online

`https://www.hpc.ntnu.no/vilje/software/mpi-and-mpi-io-training-tutorial/`

under **Basic MPI** the hyperlink to the text  Job Queue points to a page which doesn't  seem to exist

and @<!-- -->rafabench in `#449` clearly wanted to add a reference

Reply:
> see text of the closed pull for my initial reasoning and the tree command output

I can't say that was particularly clear to me.  How about linking https://www.hpc.ntnu.no/vilje/software/mpi-and-mpi-io-training-tutorial/basic-mpi/job-queue/ instead?  That seems to be the up-to-date page you actually wanted to link.

Reply:
> clear

it might have been indicated to at least identifiy the extracted .tar.gz subdirectory with the pertinent files

thank you for tracking down the current webpage address

the therein rendered `src/job_queue.c` file's `#include` lines don't have headers

and the page's hyperlink on the text code (obtain all the example[s]) is wrong

my preference is to take time with this

webpage has more info than just the code

the .tar.gz extraction has more subdir pertinent files than just the webpage's rendered .c file

Reply:
I left the top level webpage in place for contextual reasons.

Is the revision clear at this point or do you suggest any other modifications?

Reply:
I like the clarification above, but I still don't like the random wget line: it isn't super clear what's for, and wget isn't even necessarily a good suggestion, since it isn't available by default in many systems. I'd say that continuing the text above with something like 
```suggestion
# of https://www.hpc.ntnu.no/wp-content/uploads/2019/09/mpiexamples.tar.gz
```
would work for me.

Reply:
I guess this newest revision is concise.

Incidentally in `.github/workflows/UnitTests.yml` there are two `wget` lines.

Reply:
> Incidentally in `.github/workflows/UnitTests.yml` there are two `wget` lines.

I'm not sure how that's any relevant here:

1. it isn't user-facing documentation
2. that's for running code in controlled environment where we do know `wget` is available